pipelines/skylab/multiome/Multiome.wdl

```diff
@@ -124,7 +125,8 @@ workflow Multiome {
             atac_nhash_id = atac_nhash_id,
             adapter_seq_read3 = adapter_seq_read3,
             atac_expected_cells = expected_cells,
-            peak_calling = run_peak_calling
```

Comment:
why not use the top level input here?

Reply:
The ATAC workflow has peak calling built in, so if we set it to top level here, it will run at this step. But we don't want the peak calling to run yet. We want to run peak calling as a separate task after JoinBarcodes so that we can use the JoinBarcodes output.